cudamapper/CMakeLists.txt

```diff
@@ -13,6 +13,22 @@ project(cudamapper)
 # Process data subdirectory first
 add_subdirectory(data)
 
+execute_process(COMMAND
```

Comment:
let's move the section which generates the git based version number into `$TOP/cmake/GitVersion.cmake`